.github/renovate.json5

```diff
@@ -1,110 +1,109 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:best-practices",
-    ":semanticCommitsDisabled"
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+  extends: [
+    'config:best-practices',
+    ':semanticCommitsDisabled',
   ],
-
-  "branchPrefix": "grafanarenovatebot/",
-  "commitMessagePrefix": "Chore: ",
-  "platformCommit": true,
-  "dependencyDashboard": false,
-  "forkProcessing": "disabled",
-  "rebaseWhen": "behind-base-branch",
-  "prHourlyLimit": 0,
-  "prConcurrentLimit": 10,
-  "branchConcurrentLimit": 10,
-
-  "enabledManagers": ["custom.regex", "gomod"],
-
-  "labels": ["dependencies"],
-
-  "gomod": {
-    "enabled": true
+  branchPrefix: 'grafanarenovatebot/',
+  commitMessagePrefix: 'Chore: ',
+  platformCommit: 'enabled',
```

Comment:
This (true -> enabled) is the actual change in this commit.

The last time it tried to do this, it broke.

Admittedly, the last time it came from the stubborn bot that doesn't understand that we don't want it here.